### PR TITLE
Logging (on console and into file)

### PR DESCRIPTION
**Pull request by tradej**
Resolves RHOARDOC-1140

- [x] Have you followed the guidelines in our [Contributing](http://wildfly-swarm.io/community/contributing/) document?
- [x] Have you created a [JIRA](https://issues.jboss.org/browse/SWARM) and used it in the commit message?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/wildfly-swarm/wildfly-swarm/pulls) for the same issue?
- [x] Have you built the project locally prior to submission with `mvn clean install`?

-----

This is basically a modular re-hash of the original logging section in the User Guide with a few bits thrown in.

@inoxx03, @rhoads-zach, would you mind to give this a quick once-over from the technical writing perspective? Thanks. Note: This is general logging content, debugging per se will be a separate chapter pointing to this.

### CODE REVIEW

**Review by heiko-braun — APPROVED** [review]

File: docs/howto/enabling-logging/index.adoc [hunk 1]
```diff
@@ -0,0 +1,35 @@
+
+[#enabling-logging_{context}]
+= Enabling logging
+
+To enable logging, add the Logging fraction to the POM file of your application.
```
Comment (Ladicek) [inline docs/howto/enabling-logging/index.adoc:5]: Actually logging is always added :-) I mean -- it's a dependency of all fractions, so the moment you add any fraction, logging is added transitively.

File: docs/howto/logging-to-a-file/index.adoc [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+.Prerequisites
+
+* A Maven-based application with the logging fraction enabled. For more information, see xref:enabling-logging_{context}[].
+* A writable directory on your file system.
+
+.Procedure
+
+. Open a YAML profile file of your choice.
+If you do not know which one to use, open `project-defaults.yml` in the root directory of the WAR archive with your application.
```
Comment (Ladicek) [inline docs/howto/logging-to-a-file/index.adoc:19]: I don't think we'd recommend modifying WAR files directly. Typically, in a Maven project, these config files are located in `src/main/resources`.

**Comment by tradej** [comment]
Thanks for the remarks, @Ladicek, I'll file a follow-up request.

**Comment by Ladicek** [comment]
Thanks @tradej.

File: docs/howto/enabling-logging/index.adoc (hunk 1 above)
Comment (ghost) [reply docs/howto/enabling-logging/index.adoc:5]: You could change this to:
> To enable logging, verify the Logging fraction has been added to the POM file of your application.

File: docs/howto/enabling-logging/index.adoc [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+This automatically enables logging to the console on the `INFO` level and higher.
+
+.Prerequisites
+
+* A Maven-based application
+
+.Procedure
+
+. Find the `<dependencies>` section in the `pom.xml` file of your application.
+Add to it the following coordinates:
```
Comment (ghost) [inline docs/howto/enabling-logging/index.adoc:15]: Change this to
>Verify it contains:

File: docs/howto/logging-to-a-file/index.adoc [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+  FILE:
+    file:
+      path: target/MY_APP.log
+    suffix: .yyyy-MM-dd
+    named-formatter: LOG_FORMATTER
+    level: INFO
+----
+
+Here, a new handler named `FILE` is created, logging events of the `INFO` level and higher.
+The file stores logs in the `target` directory, and each log file is named `MY_APP.log` with the suffix `.yyyy-MM-dd`.
```
Comment (ghost) [inline docs/howto/logging-to-a-file/index.adoc:54]: update to:
> Logs in the `target` directory, and ...

**Review by ghost — COMMENTED** [review]
Overall LGTM outside of minor changes

**Pull request merged**